beacon_node/src/cli.rs

```diff
@@ -533,6 +533,16 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                        [default: 8192 (mainnet) or 64 (minimal)]")
                 .takes_value(true)
         )
+        .arg(
+            Arg::with_name("epochs-per-migration")
+                .long("epochs-per-migration")
+                .value_name("N")
+                .help("The number of epochs to wait between running the migration of data from the \
+                       hot DB to the cold DB. Less frequent runs can be useful for minimizing disk \
+                       writes")
```

Comment:
I guess the downside of setting this feature too high (e.g., higher than 8096) would be that we start doing state reads from the hybrid iterators because we don't have the linear history in the cold DB yet?

Reply:
Yeah, I think performance might not be great with super long durations. The reasonable max would be around 8192 slots / 32 = 256 epochs I think.

I know @ariskk was interested in a mode where Lighthouse stores more of the recent history in the state DB, so maybe we can ask him to test it out and inform us of performance deficits. I suspect just using tree-states with all states reconstructed will be faster and more disk efficient